bindings/matrix-sdk-crypto-ffi/src/backup_recovery_key.rs

```diff
@@ -63,6 +63,20 @@ pub struct MegolmV1BackupKey {
     pub backup_algorithm: String,
 }
 
+#[derive(uniffi::Record)]
+pub struct EncryptedSessionData {
+    /// Unpadded base64-encoded public half of the ephemeral key.
+    pub ephemeral: String,
+    /// Ciphertext, encrypted using AES-CBC-256 with PKCS#7 padding, encoded in base64.
+    pub ciphertext: String,
+    /// First 8 bytes of MAC key, encoded in base64.
+    pub mac: String,
+    /// v2 MAC of the backup data
+    // FIXME: should we be passing the entire `unsigned` field?  Also, how do
+    // we handle other properties?
```

Comment:
should we be passing the entire `unsigned` field?  Also, how do we handle other properties?